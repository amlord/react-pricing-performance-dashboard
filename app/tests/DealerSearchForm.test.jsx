const React = require('react');
const renderer = require('react-test-renderer');

const DEALERS = require('../api/dealer-data');
const DealerSearchForm = require('../components/dealer-search/form/DealerSearchForm');

test('Snapshot: Dealer Search Form', () =>
{
    const SEARCH_FORM = renderer.create(
        <DealerSearchForm
            searchTerm="ABC" />
    );

    let tree = SEARCH_FORM.toJSON();
    expect(tree).toMatchSnapshot();
});