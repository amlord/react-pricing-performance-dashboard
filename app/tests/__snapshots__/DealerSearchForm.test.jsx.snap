// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot: Dealer Search Form 1`] = `
<div
  className="dealerMap__searchFormContainer"
>
  <form
    className="dealerMap__searchForm"
    onSubmit={[Function]}
  >
    <input
      autoFocus={true}
      className="dealerMap__searchText"
      onChange={[Function]}
      placeholder="Search dealer name or code"
      value=""
    />
  </form>
</div>
`;
